source/docs/software/commandbased/convenience-features.rst

```diff
@@ -1,288 +1,341 @@
 Convenience features
 ====================
 
-While the previously-described methodologies will work fine for writing
-command-based robot code, the command-based libraries contain several
-convenience features for more-advanced users that can greatly reduce the
-verbosity/complexity of command-based code. It is highly recommended
-that users familiarize themselves with these features to maximize the
-value they get out of the command-based libraries.
-
-.. _inlined-commands:
+While the previously-described methodologies will work fine for writing command-based robot code, the command-based libraries contain several convenience features for more-advanced users that can greatly reduce the verbosity/complexity of command-based code. It is highly recommended that users familiarize themselves with these features to maximize the value they get out of the command-based libraries.
 
 Inline command definitions
 --------------------------
 
-While users are able to create commands by explicitly writing command
-classes (either by subclassing ``SendableCommandBase`` or implementing
-``Command``), for many commands (such as those that simply call a single
-subsystem method) this involves a lot of wasteful boilerplate code. To
-help alleviate this, many of the prewritten commands included in the
-command-based library may be *inlined* - that is, the command body can
-be defined in a single line of code at command construction.
+While users are able to create commands by explicitly writing command classes (either by subclassing ``CommandBase`` or implementing ``Command``), for many commands (such as those that simply call a single subsystem method) this involves a lot of wasteful boilerplate code. To help alleviate this, many of the prewritten commands included in the command-based library may be *inlined* - that is, the command body can be defined in a single line of code at command construction.
 
 Passing subroutines as parameters
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-In order to inline a command definition, users require some way to
-specify what code the commands will run as constructor parameters.
-Fortunately, both Java and C++ offer users the ability to pass
-subroutines as parameters.
+In order to inline a command definition, users require some way to specify what code the commands will run as constructor parameters. Fortunately, both Java and C++ offer users the ability to pass subroutines as parameters.
```

Comment:
"users require some way"
This wording confuses me? Is it the user requiring this, or is it required of the users?